src/metadata.mjs

```diff
@@ -109,7 +109,7 @@ const createMetadata = slugger => {
      * The Navigation entries has a dedicated separate method for retrieval
      * as it can be manipulated outside of the scope of the generation of the content
      *
-     * @param {import('vfile').VFile} apiDoc The API doc file being parsed
+     * @param {{stem?: string, basename?: string}} apiDoc The API doc file being parsed
```

Comment:
Isn't this still a VFile?

Reply:
Yeah, it is but some properties like `stem` and `basename` are not enumerable in VFile, so I needed to serialize them before sending to worker threads.

Reply:
IMO, but let's see what others think, we should use the VFile type when possible, since we have it.